<?xml version="1.0" ?>
<launch>

  <arg name="sim"			default="false"/>
  <arg name="camera_name"		default="realsenseD435"/>
  <arg name="check"			default="false"/>
  <arg name="scene"			default=""/>
  <arg name="use_dual_quaternion"	default="true"/>

  <!-- Set camera topic names -->
  <arg name="camera_info_topic"		value="color/camera_info"/>
  <arg name="image_topic"		value="color/image_raw"/>
  <arg name="depth_topic"		value="aligned_depth_to_color/image_raw"/>

<!-- start realsenseD435 alignedDepth:true -->
	<!-- <include file="$(find realsense2_camera)/launch/rs_rgbd.launch">
    <arg name="align_depth"  default="true"/>
  </include> -->

  <!-- Launch devices and calibrator -->
  <include file="$(dirname)/include/handeye_calibration.launch">
    <arg name="config"			value="mocap"/>
    <arg name="scene"			value="$(arg scene)"/>

    <arg name="camera_info_topic"	value="$(arg camera_info_topic)"/>
    <arg name="image_topic"		value="$(arg image_topic)"/>
    <arg name="depth_topic"		value="$(arg depth_topic)"/>

    <arg name="sim"			value="$(arg sim)"/>
    <arg name="camera_name"		value="$(arg camera_name)"/>
    <arg name="check"			value="$(arg check)"/>
    <arg name="use_dual_quaternion"	value="$(arg use_dual_quaternion)"/>
  </include>

</launch>
